.github/workflows/go-check.yml

```diff
@@ -15,4 +15,6 @@ concurrency:
 
 jobs:
   go-check:
-    uses: pl-strflt/uci/.github/workflows/go-check.yml@v0.0
+    uses: pl-strflt/uci/.github/workflows/go-check.yml@release-v0.0.13
```

Comment:
This is just testing before release. We should restore v0.0 once v0.0.13 is out.

```suggestion
    uses: pl-strflt/uci/.github/workflows/go-check.yml@v0.0
```

Reply:
So should we merge this as is or wait for the release?

Reply:
We can go back to this now:
```suggestion
    uses: pl-strflt/uci/.github/workflows/go-check.yml@v0.0
```